SubscribeIT/ngDesk
# 14 unable to add read only fields

#14 

##### Test Case 1
Name :- Create layouts

1. Login to ngdesk.
2. Goto -> modules -> tickets -> layouts -> create layouts -> new.
3. Fill details.
4. Drag not editable fields.
5. Drag other fields.
6. Hit save.
7. Observe getting saved.
8. Observe no error.


##### Test Case 2
Name :- Edit layouts

1. Login to ngdesk.
2. Goto -> modules -> tickets -> layouts -> edit layouts -> new.
3. Fill details.
4. Drag not editable fields and observe the error message.
5. Drag fields to header.
6. Observe the header fields.
7. Hit save.
8. Observe getting saved.
9. Observe no error.

##### Test Case 3
Name :- Edit layouts (update program)

1. Login to ngdesk.
2. Goto -> modules -> tickets -> layouts -> edit layouts -> click on layout created above.
3. Run update program.
4. Refresh the page
5. Observe the delete button is not there.
6. Observe no error.

**screen-shots**

![Screenshot from 2021-08-30 16-18-07](https://user-images.githubusercontent.com/89504119/131328272-ddbed558-875b-404c-b598-3956162cac51.png)
